app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -44,7 +77,124 @@ fun MapScreen(navigationActions: NavigationActions) {
         MapboxMap(
             Modifier.fillMaxSize().padding(padding).testTag("MapScreen"),
             mapViewportState = mapViewportState,
-            style = { MapStyle("mapbox://styles/seanprz/cm27wh9ff00jl01r21jz3hcb1") })
+            style = { MapStyle("mapbox://styles/seanprz/cm27wh9ff00jl01r21jz3hcb1") }) {
+              DisposableMapEffect { mapView ->
+
+                // Set camera bounds options
+                val cameraBoundsOptions =
+                    CameraBoundsOptions.Builder().minZoom(8.0).maxZoom(18.0).build()
```

Comment:
You created const values for min zoom and maxzoom but didn't use them here

Reply:
Fixed !